package.json

```
@@ -67,8 +67,12 @@
     "build-storybook": "build-storybook",
     "prestart-storybook": "yarn storybook-load-new",
     "start-storybook": "STORYBOOK=1 yarn start",
-    "detect-secrets-staged": "scripts/detect-secrets-staged",
-    "detect-secrets-all": "scripts/detect-secrets-all",
+    "secrets-audit": "detect-secrets audit .secrets.baseline",
+    "secrets-check-staged": "scripts/secrets-check-staged",
+    "secrets-check-all": "scripts/secrets-check-all",
+    "secrets-add-staged": "scripts/secrets-add-staged",
+    "secrets-add-all": "scripts/secrets-add-all",
+    "secrets-generate-once": "scripts/secrets-generate-baseline",
```

```suggestion
    "secrets:audit": "detect-secrets audit .secrets.baseline",
    "secrets:check:staged": "scripts/secrets-check-staged",
    "secrets:check:all": "scripts/secrets-check-all",
    "secrets:add-from:staged": "scripts/secrets-add-staged",
    "secrets:add-from:all": "scripts/secrets-add-all",
    "secrets:generate!": "scripts/secrets-generate-baseline",
```

can we have `!` at the end of a script? let me try locally